package.json

```diff
@@ -9,7 +9,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10"
+    "node": ">=14.0.0"
```

Comment:
Rather than change this, either remove it altogether or leave it as-is and remove it in a separate PR. engines refers to production operational requirements, and while this app does need Node to compile translations, that's a build requirement not a production requirement since this app runs in a browser.

Unfortunately, there's not a way to indicate build-time operational requirements. Oh well.